packages/sw-precaching/src/lib/controllers/base-cache-manager.js

```diff
@@ -13,7 +13,8 @@ class BaseCacheManager {
    */
   constructor(cacheName) {
     this._entriesToCache = new Map();
-    this._cacheName = cacheName;
+    this._requestWrapper = new RequestWrapper({cacheName});
```

Comment:
```js
this._requestWrapper = new RequestWrapper({
  cacheName,
  fetchOptions: {
    credentials: 'same-origin',
  },
});
```

gives you the same [`fetch` options](https://github.com/GoogleChrome/sw-helpers/blob/master/packages/sw-runtime-caching/src/lib/request-wrapper.js#L50) that the previous code (and like `sw-precache`) uses.